base/src/test/java/uk/ac/ebi/atlas/solr/analytics/fullanalytics/AnalyticsQueryBuilderTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi.atlas.solr.analytics.fullanalytics;
+
+import org.apache.solr.client.solrj.SolrQuery;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.isA;
+import static org.junit.Assert.assertThat;
+
+public class AnalyticsQueryBuilderTest {
```

Comment:
A quick test that would nevertheless be verifying something and not giving false positives would be when set a field value to be a magic value - say a cutoff is 133.7 -  and assert it appears somewhere in the result as a substring.